### ISSUE TITLE
Tabs again

### ISSUE THREAD

**Issue by vojtechkral** [vmg/redcarpet#496]
Follow up to #153 

1) Is there really a reason why tabs need to be converted to spaces during preprocessing? I'm not sure I really understand the reasoning, especially when it comes to code blocks.

2) If it really is necessary to replace tabs, could an option be provided that sets how many spaces per tab to replace?

**Comment by joesavage** [comment]
I too would love a tab-preserving option, regardless of the limitations it may place on the indentation requirements elsewhere in the file. I'm very particular about the types of indentation I use (typically, a mix of tabs and spaces for different circumstances), and so this is a huge paint point for me.

**Comment by vojtechkral** [comment]
Yeah, plus there are cases where you _have_ to use tabs, such as Makefiles. As of now, inserting syntactically correct Makefiles is not possible.

**Comment by rmzelle** [comment]
Same here.

I want to show a code block with tab-delimited content as an example of how users should format their data (in GitHub flavored markdown, which seems to be processed with redcarpet), and the conversion of tabs to spaces kinda defeats the purpose of this.

**Comment by vojtechkral** [comment]
Ok, well, I'm switching to kramdown. It's not out of spite or anything, I just need tabs...
